pkg/controller/helmrelease/helmrelease_controller.go

```diff
@@ -605,3 +336,331 @@ func hasHelmUpgradeForceAnnotation(hr *appv1.HelmRelease) bool {
 	}
 	return value
 }
+
+func (r *ReconcileHelmRelease) install(instance *appv1.HelmRelease, manager helmoperator.Manager) (reconcile.Result, error) {
+	if instance.Status.DeployedRelease != nil {
+		log.Info("Release is not installed but status.DeployedRelease is populated. Possible Helm storage corruption")
```

Comment:
why are we using 2 loggers? here we are using `log` but later we are using `klog`